.eslintrc

```diff
@@ -26,7 +26,7 @@
     "react/no-find-dom-node": 2,
     "react/forbid-prop-types": 0,
     "react/no-danger": 0,
-    "react/no-children-prop": 0,
+    "react/no-children-prop": 2,
```

Comment:
Instead of explicitly enabling this, we should just remove the line that disables it.

The baseline rules come from the eslint modules in package.json, and new rules get added or removed when we update those packages.

Reply:
You could also remove the no-find-dom-node line; I should have noted the same thing when that change got merged, but I didn't think of it.

Reply:
Ok, I've done it here and on other branches :).